src/blister.c

```diff
@@ -284,17 +284,31 @@ Obj DoCopyBlist(Obj list, Int mut) {
   
 }
 
+Obj CopyBlistImm (
+    Obj                 list,
+    Int                 mut )
+{
+  return list;
+}
+		  
 Obj CopyBlist (
     Obj                 list,
     Int                 mut )
 {
 
-    /* don't change immutable objects                                      */
-    if ( ! IS_MUTABLE_OBJ(list) ) {
-        return list;
-    }
-
-    return DoCopyBlist(list, mut);
+  Obj copy;
+  Obj tmp;
+ 
+    copy =  DoCopyBlist(list, mut);
```

Comment:
Why is the declaration of the locals indented differently than the rest of the code? (For that matter: The other function you introduce indent by 2 spaces, this function by 4, mostly)